edc-extensions/dataplane/dataplane-self-registration/src/main/resources/META-INF/services/org.eclipse.edc.spi.system.ServiceExtension

```diff
@@ -0,0 +1,20 @@
+#################################################################################
+#  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Comment:
```suggestion
#  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Reply:
Nice catch! Changed in https://github.com/eclipse-tractusx/tractusx-edc/pull/1498/commits/040a5029507945d6671eeef243ad37647183e9ee